embedchain/embedchain.py

```diff
@@ -50,6 +51,8 @@ def __init__(
         :type system_prompt: Optional[str], optional
         :raises ValueError: No database or embedder provided.
         """
+        # Setup user directory if it doesn't exist already
+        Client.setup_dir()
```

Comment:
I think this needs to be outside the Embedchain class to be able to run when we do `import embedchain`